apps/web/pages/signup.tsx

```diff
@@ -181,18 +181,27 @@ export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
     };
   }
 
-  const verificationToken = await prisma.verificationToken.findUnique({
+  const verificationToken = await prisma.verificationToken.findFirst({
     where: {
       token,
+      expires: {
+        gte: new Date(),
+      },
     },
   });
 
-  if (!verificationToken || verificationToken.expires < new Date()) {
+  if (!verificationToken) {
     return {
       notFound: true,
     };
   }
 
+  await prisma.verificationToken.deleteMany({
+    where: {
+      token,
+    },
+  });
```

Comment:
I think this can be just delete 

token is unique in our DB so this should never be many